plugin.program.utorrent/default.py

```diff
@@ -21,7 +21,9 @@
 UT_PASSWORD = __addon__.getSetting('pwd')
 UT_TDIR = xbmc.translatePath( __addon__.getSetting('tdir') )
 UT_HTTPS = __addon__.getSetting('use_https')
-UT_PATH = __addon__.getSetting('path')
+UT_PATH = '/' + __addon__.getSetting('path').strip('/') + '/'
+if UT_PATH != __addon__.getSetting('path'):
```

Comment:
Yes, line 24 makes sure that path is correct, but updating path to correct one on settings view may be helpful if there is some problems with connecting to utorrent. It's better if user sees what settings are actually used.